app/views/support/BillingPeriod.scala

```diff
@@ -8,6 +8,7 @@ object BillingPeriod {
       case Month => "every month"
       case Quarter => "every 3 months"
       case Year => "every 12 months"
+      case SixMonthsRecurring => "every six months"
```

Comment:
Presumably these people do still pay every 6 months (just via some manual process) so I think it makes sense?